### DISCUSSION TITLE
Unraid with Mealie beta

### DISCUSSION THREAD

**Discussion by DrSpaldo** [mealie-recipes/mealie#2236] [Q&A] [answered]
I was wondering if anyone has been using the Mealie beta branch with Unraid? I wasn't sure which one to put as the Repository. There appears to be quite a few different tags; omni-nightly, frontend-nightly, api-nightly, frontend-v1.0.0beta-5, latest, etc.

**Comment by coffee-mage** [comment, marked as answer]
Hi DrSpaldo,

The original docker structure had the api and the frontend broken into two separate containers.  That is why you see frontend* and api*.   To make mealie work, you need both.   beta5 was the last released stable beta version  (Oct 25th, 2022).    Nightly is a straight pull from wherever the code is written the night before.

Omni-nightly was created due to popular demand, for merging together the api and frontend docker images.  This means if you run omni, you only need to use one docker.   It seems to have worked really well, and several people have switched to it  (myself included).

The only challenge with omni-nightly is that you are on the nightly codebase.   I have been using the nightly codebase in production for about a month.  I only had one issue, a bug nuked 2 recipes.  But I restored from a backup.    For me the benefit of using omni-nightly outweighs the drawbacks (one container,  significantly newer code with alot of improvements).

If I had to make a recommendation, I would say omni-nightly, but don't update the container every day.  ;)    Also I would strongly recommend that you use Unraid's  plugin CA Backup/Restore AppData.   Have it back up every few days.  If you get hit with some bug, you can always restore.

Good luck!  Drop by discord if you want to chat in more detail.

**Reply by swails** [reply]
Is omni-nightly updated more frequently than frontend-nightly/api-nightly?

I deployed the latter (and would like to be regular with updates), but will switch to the former if the latter isn't kept up to date as much.

**Reply by DrSpaldo** [reply]
> Hi DrSpaldo,
> 
> The original docker structure had the api and the frontend broken into two separate containers. That is why you see frontend* and api*. To make mealie work, you need both. beta5 was the last released stable beta version (Oct 25th, 2022). Nightly is a straight pull from wherever the code is written the night before.
> 
> Omni-nightly was created due to popular demand, for merging together the api and frontend docker images. This means if you run omni, you only need to use one docker. It seems to have worked really well, and several people have switched to it (myself included).
> 
> The only challenge with omni-nightly is that you are on the nightly codebase. I have been using the nightly codebase in production for about a month. I only had one issue, a bug nuked 2 recipes. But I restored from a backup. For me the benefit of using omni-nightly outweighs the drawbacks (one container, significantly newer code with alot of improvements).
> 
> If I had to make a recommendation, I would say omni-nightly, but don't update the container every day. ;) Also I would strongly recommend that you use Unraid's plugin CA Backup/Restore AppData. Have it back up every few days. If you get hit with some bug, you can always restore.
> 
> Good luck! Drop by discord if you want to chat in more detail.

Thanks so much for the reply & explanation @braddillon . I have been looking around for the difference between the builds and didn't see anything obvious. Did you do anything specific to get the beta running on Unraid? or can I just change the repository to the omni and it should work?

I will look later for the Discord link and jump in regardless to say hi

**Reply by coffee-mage** [reply]
Both omni-nightly and frontend/api-nightly are updated with the same frequency  (every night).

**Reply by coffee-mage** [reply]
Hi Dr.
Nothing special for setup,  here are my settings:
<img width="1132" alt="mealie_config" src="https://user-images.githubusercontent.com/14890934/224780971-dd5fa7bc-bcaa-4f64-a3f9-c9161a25b9a2.png">

**Comment by DrSpaldo** [comment]
Great, thanks @braddillon , I will make mine more like yours. I think I added too many ports (although it does seem to be working)

**Comment by DrSpaldo** [comment]
For those that may come across this thread, I ended up having to change the repository tag to:

ghcr.io/mealie-recipes/mealie:nightly

There is also a change in the port/s. Now it is just 9000. 

To be easier (for me). I did a backup, deleted my old docker and re-added the app via the Unraid app store. That was the way I did it. However,  I am sure those that are smarter may be able to make some other changes to the current setup. ...
